frontend/src/features/collections/collection-thumbnail.ts

```diff
@@ -0,0 +1,52 @@
+import { localized } from "@lit/localize";
+import { html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+import { BtrixElement } from "@/classes/BtrixElement";
+import thumbnailCyanSrc from "~assets/images/thumbnails/thumbnail-cyan.avif";
+import thumbnailGreenSrc from "~assets/images/thumbnails/thumbnail-green.avif";
+import thumbnailOrangeSrc from "~assets/images/thumbnails/thumbnail-orange.avif";
+import thumbnailYellowSrc from "~assets/images/thumbnails/thumbnail-yellow.avif";
+
+export enum Thumbnail {
+  Cyan = "thumbnail-cyan",
+  Green = "thumbnail-green",
+  Orange = "thumbnail-orange",
+  Yellow = "thumbnail-yellow",
+}
+
+export const DEFAULT_THUMBNAIL = Thumbnail.Cyan;
+
+@localized()
+@customElement("btrix-collection-thumbnail")
+export class CollectionThumbnail extends BtrixElement {
+  static readonly Variants: Record<Thumbnail, { path: string }> = {
+    [Thumbnail.Cyan]: {
+      path: thumbnailCyanSrc,
+    },
+    [Thumbnail.Green]: {
+      path: thumbnailGreenSrc,
+    },
+    [Thumbnail.Orange]: {
+      path: thumbnailOrangeSrc,
+    },
+    [Thumbnail.Yellow]: {
+      path: thumbnailYellowSrc,
+    },
+  };
+
+  @property({ type: String })
+  src?: string;
+
+  render() {
+    return html`
+      <img
+        class="aspect-video rounded-lg border border-cyan-100 bg-slate-50 object-cover"
```

Comment:
I would go for a neutral border & shadow here, rather than cyan — I think it clashes a little with non-neutral or cyan colors in thumbnails
<img width="467" alt="Screenshot 2024-12-17 at 3 42 27 PM" src="https://github.com/user-attachments/assets/d8ab2b09-fc01-4c71-9736-865a65642b97" />

Reply:
This is also a lot more subtle, but when combining a border and a shadow, I find it tends to work a lot better to use a Tailwind `ring` with a semi-transparent dark tint, rather than a solid color — this way, you'll get consistent contrast between the ring and the shadow on all sides. We made use of this a lot on the website (see the second screenshot).
<img width="698" alt="Screenshot 2024-12-17 at 3 43 59 PM" src="https://github.com/user-attachments/assets/ff9c6d99-4be9-4829-8c3f-fb2bb575a78f" />
<img width="132" alt="Screenshot 2024-12-17 at 3 46 15 PM" src="https://github.com/user-attachments/assets/1a1178f7-7f4c-4b7f-9ca6-3160899a2376" />

Reply:
Good suggestion, updated!

<img width="311" alt="Screenshot 2024-12-17 at 1 53 00 PM" src="https://github.com/user-attachments/assets/eb4c95f2-8e0d-4c75-b574-0a730c96b80d" />